historical-frontend/choose.html

```diff
@@ -5,7 +5,7 @@
 
 {% include_relative _heading.html %}
 
-<h1 class="heading-medium">Choose a version to start with</h1>
```

Comment:
Hmm, I prefer this wording as it tells users what to do with the following links, otherwise it's just a list of links.